### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by batoolfatima2135**
In this project, I have :

  1. Set up a new project with webpack that is based on the webpack exercise you have already completed.
  2. Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder.
  3. Create an index.js file and set an array of simple to-do tasks (array of objects). Each task object should contain three keys:
    - description [string].
    - completed [bool].
    - index: [number].   
  7. Write a function to iterate over the tasks array and populate an HTML list item element for each task.
  8. On page load render the dynamically created list of tasks in the dedicated placeholder.
  9. Create a style.css and set rules for the To Do List. Webpack Style/CSS Loader loads CSS.

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @batoolfatima2135,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

* Kindly remove the live demo link or provide a deployed version of the app.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @batoolfatima2135, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job setting up your to-do list structure 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/display.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    label.classList.add('flex-grow-1', 'text-left', 'm-1', 'form-control');
+    label.value = task.description;
+    li.appendChild(label);
+    // Creating option icon
+    const icon = document.createElement('i');
+    icon.className = 'fa-solid fa-ellipsis-vertical';
+    li.appendChild(icon);
+    // Appending elements
+    ul.appendChild(li);
+  });
```
[the remark this answers is not in the document]
Comment (batoolfatima2135) [reply src/modules/display.js:25]: Hey @mckent05,
An array of tasks is in the index.js file. My main js file i.e.` index.js` contain an array of tasks(array of objects) with each object containing three keys - description [string], completed [bool], index: [number].  I have implemented that.  
Attaching screenshot for reference :
Capture

File: src/modules/display.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
```
Comment (mckent05) [inline src/modules/display.js]: - Good job done so far, you seem to have done everything right but the tasks are not displaying in the UI on my end, please fix this 🙏 

Your project:
35yw35y

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @batoolfatima2135, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍
- Good job creating the to-do list structure for your project 👍 👍  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @batoolfatima2135 👍, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Webpack is used to set up the project ✔️ 
Implemented structure for todo-list 👍 
Had an array of some simple *to do tasks* ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**